bydbctl/internal/cmd/root.go

```diff
@@ -96,14 +98,23 @@ func initConfig() {
 		viper.SetConfigName(".bydbctl")
 	}
 
+	viper.SetEnvPrefix("BYDBCTL")
 	viper.AutomaticEnv()
```

Comment:
You could use bindFlags from `pkg/config` to bind all flags to env vars. I'm free to make this function exported